ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -335,9 +341,10 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
         container.addArrangedSubview(label)
 
         if interactive {
+            let showCustomSubtitleImage = (accessoryType == .custom) && (container == subtitleContainer)
```

Comment:
nit: Should we factor out the two container checks to just `let isTitle = container == titleContainer` and then check `!isTitle` here and pass in `isTitle` below?

Reply:
Yep that's better I'll do that!